### DISCUSSION TITLE
Icons not showing for sketchybar

### DISCUSSION THREAD

**Discussion by goofenoff** [FelixKratz/SketchyBar#427] [Q&A]
I tried reinstalling sketchybar a bunch of times, but it just shows question marks for the icons and this black bar with information over it. Idk what im doing wrong, as I also installed the fonts a couple of times as well.

**Comment by FelixKratz** [comment]
This is a font problem. Which fonts are used by your config and which ones did you install? There are various configs around that require additional fonts e.g. SF Symbols.

Also note that the nerdfonts did a breaking change with their v3.0 which some of the configs did not yet migrate to.

Since you say you have a black bar, you are probably not using the default config.

**Comment by goofenoff** [comment]
yeah i wasnt using default config, my bad 😂, im still new to this and i definitely made a mistake somewhere, ran the default config, but it still shows the same question marks, i installed hack nerd font

**Comment by Shyboy0499** [comment]
Sorry, but I really don't know how to change the font for sketchybar, also I am using iTerm as terminal, and iTerm's icons is missing as well, so I think that the font for the system is not the regular font. 
![Uploading 截屏2024-04-10 20.30.37.png…]()

**Reply by Nikolaidp24** [reply]
You need [sketchybar-app-font](https://github.com/kvndrsslr/sketchybar-app-font) for the icons to work properly, it's custom font made for SketchyBar. Also in your config files, you can specify which font to use for which module, or just name a font for general use. You can check the [guide](https://felixkratz.github.io/SketchyBar/config/bar) on how to set them.

**Reply by Kryckan** [reply]
This seems to be an issue still as in the installation instructions with the default config, it is specified to use hack nerd font and it still is not showing the icons. So either the instructions need to be updated or something has happened on Apples side?
